File: leet_1662_check_same_strings.js
/**
 * @param {string[]} word1
 * @param {string[]} word2
 * @return {boolean}
 */



var arrayStringsAreEqual = function(word1, word2) {
    let  s1 = "", s2 = "";
    for (let w of word1)
        s1 += w;
    for (let w of word2)
        s2 += w;
    return (s1 === s2);
};

File: leet_1662_check_same_strings.go
func arrayStringsAreEqual(word1 []string, word2 []string) bool {
    s1 := ""
    s2 := ""
    for _, s := range word1 {
        s1 += s
    }
    for _, s := range word2 {
        s2 += s
    }
    return s1 == s2
}
